### DISCUSSION TITLE
Create an untitled window on re-activation only

### DISCUSSION THREAD

**Discussion by ir0nh34d** [macvim-dev/macvim#1482] [Ideas]
I have MacVim starting at login and I have set "Open Untitled Window" set to never so that it doesn't create a window at startup.  I would like to be able to click on the MacVim icon in the dock and have it open a new window (as opposed to having to select "New Window" from the menu).  So much like the option "only when MacVim launches" it would be nice to have an option "only when MacVim is re-activated".

**Comment by ychin** [comment]
Hmm, I guess it does make sense to add this.  Just curious though, why launch MacVim on boot, if you don't plan to use it? MacVim itself (rather than the Vim window itself) shouldn't take more than a second to launch I think?

**Reply by ir0nh34d** [reply]
That’s a valid point - I guess there is no benefit having MacVim in my startup items if I don’t want an initial window.  I like having MacVim at the ready as I use it frequently, but as you say, it doesn’t take long to load.

**Reply by ychin** [reply]
That's fair. I'll look into adding that. There should already be an issue created for this.

**Reply by ychin** [reply]
This is now implemented and will come out in the next release.

**Comment by ychin** [comment]
[Release 181](https://github.com/macvim-dev/macvim/releases/tag/release-181) is now out and has this feature.
